v2/README.md

```diff
@@ -221,9 +222,9 @@ func ExampleParseOperationNameAndType() {
 			continue
 		}
 		operationCount++
-		name := document.RootOperationTypeDefinitionNameString(node.Ref)
+		name := document.OperationDefinitionNameString(node.Ref)
 		operationNames = append(operationNames, name)
-		operationType := document.RootOperationTypeDefinitions[node.Ref].OperationType
```

Comment:
`RootOperationTypeDefinitions` is an empty slice here. Likewise, it wasn't clear to me whether this is intentional or not.